specification/storagesync/resource-manager/readme.java.md

```diff
@@ -16,7 +16,7 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-storagesync
 
 ``` yaml $(java) && $(multiapi)
 batch:
-  - tag: package-2019-02-01
+  - tag: package-2018-10-01
```

Comment:
Will it contain just 2018-10-01 or both 2018-10-01 and 2019-02-01 when specifying --multiapi?

Reply:
both

Reply:
You should not remove - tag: package-2019-02-01 if need both

Reply:
@iscai-msft, @ankushbindlish2 , would you take a look my comment?

Reply:
I think I got confused. We need 1 SDK version at a time. So, We just need a latest version which would be used to generate SDK , just like C#

Reply:
The behavior for Java and C# SDK are different, currently C# doesn't support multiapi.
With latest version of Java sdk, what users expect is to be able to use different API version (with different namespaces) without switching to previous version of java sdk.
If you do not plan to support multiapi, you should remove multiapi to avoid confusion.